boards/DCM/Src/App/states/App_DriveState.c

```diff
@@ -33,6 +34,8 @@ static void DriveStateRunOnTick100Hz(struct StateMachine *const state_machine)
     {
         App_SharedStateMachine_SetNextState(state_machine, App_GetInitState());
     }
+    App_SetPeriodicCanSignals_Imu(world);
```

Comment:
nit; extra space for legibility
```suggestion

    App_SetPeriodicCanSignals_Imu(world);
```